package.json

```diff
@@ -24,7 +24,7 @@
     "@babel/preset-env": "^7.13.12",
     "@babel/preset-react": "^7.12.13",
     "@babel/preset-typescript": "^7.13.0",
-    "@centreon/frontend-core": "github:centreon/frontend-core",
+    "@centreon/frontend-core": "github:centreon/frontend-core#6079299b5e4b7608813bfa40df90f724ace6bbe9",
```

Comment:
Will you take off this commit hash in another PR ?

Reply:
Yes, I’ll wait that all the current PRs are merged so that we don’t deal with gigantic conflicts